.env.docker

```diff
@@ -145,4 +146,13 @@ TRUST_PROXIES="*"
 
 ## Passport
 #PASSPORT_PRIVATE_KEY=
-#PASSPORT_PUBLIC_KEY=
\ No newline at end of file
+#PASSPORT_PUBLIC_KEY=
+
+# OIDC for logins
+OIDC_CLIENT_ID=pixelfed
+OIDC_CLIENT_SECRET=F55hjj2FBPnnuW7nD80LjwS9sVYXm4fB
```

Comment:
I'm sure this is just for testing.. but I'd probably remove these from the PR

Reply:
removed! I've also restored the `.env.docker` to its default (empty) form